src/components/app/app-authenticated/app-authenticated.tsx

```diff
@@ -12,11 +12,13 @@ import Home from '../../home/home';
 import NavTopBar from '../../whole-app/nav-top-bar/nav-top-bar';
 import Settings from '../../settings/settings';
 import styles from './app-authenticated.module.css';
+import Sidebar from '../../whole-app/sidebar/sidebar';
 
 const AppAuthenticated: React.FC = () => {
   return (
     <>
       <NavTopBar />
+      <Sidebar />
```

Comment:
you might consider wrapping sidebar and the thing to its right in a div to help better organize stuff. HTML is intended to be vertical top down so when you do something left right you typically do that inside of a element to show top down _then_ left right